.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: scala
+scala:
+   - 2.11.8
+#   - 2.12.0-M4
```

Comment:
The openjdk 8 is not available yet on travis, I can remove this if you don't like the dead code